### PR TITLE
Remove webflux 

### PR DESCRIPTION
**Pull request by archendel**

### CODE REVIEW

File: src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisWorkerService.java [hunk 1]
```diff
@@ -252,47 +242,43 @@ public Consumer<Message<String>> consumeRun() {
...
-                            return Mono.empty();
-                        })
-                        .doFinally(s -> {
-                            futures.remove(resultContext.getResultUuid());
-                            cancelComputationRequests.remove(resultContext.getResultUuid());
-                            runRequests.remove(resultContext.getResultUuid());
-                        })
-                        .block();
+                startTime.set(System.nanoTime());
+                try {
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisWorkerService.java:246]: Use the same template as other computations  for try/catch into remove the code smell

File: src/main/java/org/gridsuite/securityanalysis/server/util/SecurityAnalysisException.java [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
 public class SecurityAnalysisException extends RuntimeException {
     public enum Type {
-        RESULT_NOT_FOUND
+        RESULT_NOT_FOUND,
+        RUN_AS_ERROR
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/securityanalysis/server/util/SecurityAnalysisException.java:8]: Is used ?

File: src/test/java/org/gridsuite/securityanalysis/server/service/ReportServiceTest.java [hunk 3]
```diff
@@ -6,65 +6,52 @@
...
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
 
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
+@SpringBootTest
 @RunWith(SpringRunner.class)
+@ContextConfiguration(classes = {SecurityAnalysisApplication.class, TestChannelBinderConfiguration.class})
```
Comment (SlimaneAmar) [inline src/test/java/org/gridsuite/securityanalysis/server/service/ReportServiceTest.java:39]: Perhaps use `ContextConfigurationWithTestChannel` like study server ?

**Review by FranckLecuyer — APPROVED** [review]

File: src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisWorkerService.java (hunk 1 above)
Comment (archendel) [reply src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisWorkerService.java:246]: done

File: src/main/java/org/gridsuite/securityanalysis/server/util/SecurityAnalysisException.java (hunk 2 above)
Comment (archendel) [reply src/main/java/org/gridsuite/securityanalysis/server/util/SecurityAnalysisException.java:17]: not anymore, removing it

File: src/test/java/org/gridsuite/securityanalysis/server/service/ReportServiceTest.java (hunk 3 above)
Comment (archendel) [reply src/test/java/org/gridsuite/securityanalysis/server/service/ReportServiceTest.java:39]: done

File: src/test/java/org/gridsuite/securityanalysis/server/service/ActionsServiceTest.java [hunk 4]
```diff
@@ -40,7 +39,9 @@
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  * @author Slimane Amar <slimane.amar at rte-france.com>
  */
+@SpringBootTest
 @RunWith(SpringRunner.class)
+@ContextConfiguration(classes = {SecurityAnalysisApplication.class, TestChannelBinderConfiguration.class})
```
Comment (archendel) [inline src/test/java/org/gridsuite/securityanalysis/server/service/ActionsServiceTest.java:44]: use ContextConfigurationWithTestChannel

File: src/test/java/org/gridsuite/securityanalysis/server/service/ReportServiceTest.java (hunk 3 above)
Comment (archendel) [inline src/test/java/org/gridsuite/securityanalysis/server/service/ReportServiceTest.java:39]: use ContextConfigurationWithTestChannel

**Review by jonenst — APPROVED** [review]

File: src/test/java/org/gridsuite/securityanalysis/server/service/ActionsServiceTest.java (hunk 4 above)
Comment (archendel) [reply src/test/java/org/gridsuite/securityanalysis/server/service/ActionsServiceTest.java]: fixed

File: src/test/java/org/gridsuite/securityanalysis/server/service/ReportServiceTest.java (hunk 3 above)
Comment (archendel) [reply src/test/java/org/gridsuite/securityanalysis/server/service/ReportServiceTest.java]: fixed

**Review by SlimaneAmar — APPROVED** [review]

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**